task/rhacs-image-scan/3.71/rhacs-image-scan.yaml

```diff
@@ -49,28 +52,119 @@ spec:
 
         When set to "true", skip verifying the TLS certs of the Central endpoint and registry.
       default: "false"
+  volumes:
+    - name: token-vol
+      projected:
+        sources:
+          - serviceAccountToken:
+              audience: api
+              expirationSeconds: 3600
+              path: token
+  workspaces:
+    - name: roxctl-config
+      description: The folder where the roxctl config should be stored.
+      mountPath: /roxctl-config
+  stepTemplate:
+    env:
+      - name: HOME
+        value: /tekton/home
+      - name: INSECURE
+        value: $(params.insecure-skip-tls-verify)
+      - name: ROX_CONFIG_DIR
+        value: $(workspaces.roxctl-config.path)
+      - name: ROX_CENTRAL_ENDPOINT
+        valueFrom:
+          secretKeyRef:
+            key: rox_central_endpoint
+            name: $(params.rox_central_endpoint)
```

Comment:
IMO this probably shouldn't be a secret but just a simple string parameter, feels awkward to have to create a secret for the endpoint.

WDYT?

Reply:
I think it would indeed be better to have the endpoint passed as plain text argument rather than secret.
I suppose it was there for consistency with the ROX_API_TOKEN parameter.